src/queue.js

```diff
@@ -204,4 +204,50 @@ Queue.prototype.shutdown = function() {
   }));
 };
 
+/**
+ * Gets queue worker count
+ */
+Queue.prototype.getWorkerCount = function() {
+  return this.workers.length;
+};
+
+/**
+ * Adds a queue worker
+ */
+Queue.prototype.addWorker = function() {
+  logger.debug('Queue: adding worker');
+  var processId = (this.specId ? this.specId + ':' : '') + this.workers.length;
+  var worker = new QueueWorker(
+    this.tasksRef,
+    processId,
+    this.sanitize,
+    this.suppressStack,
+    this.processingFunction
+  );
+  this.workers.push(worker);
+
+  if (_.isUndefined(this.specId)) {
+    worker.setTaskSpec(DEFAULT_TASK_SPEC);
+  // if the currentTaskSpec is not yet set it will be called once it's fetched
+  } else if (this.currentTaskSpec) {
```

Comment:
It's not always possible to know if something might return falsy in such a loosely-typed language - in this case we don't have a lot of control over what the external dependency returns. Though it's very unlikely that the Firebase SDK will return anything that evaluates falsy for a reference, it doesn't hurt to be explicit about precisely what we want to test for (if they did, for example, that shouldn't change the behavior here).

However, even if the value were completely under our control, I think consistency should trump efficiency